counterpoll/main.py

```diff
@@ -302,11 +308,16 @@ def config_db():
 @click.argument("filename", default="/etc/sonic/config_db.json", type=click.Path(exists=True))
 def enable(filename):
     """ Enable counter configuration in config_db file """
-    _update_config_db("enable", filename)
+    _update_config_db_flex_counter_table("enable", filename)
 
 @config_db.command()
 @click.argument("filename", default="/etc/sonic/config_db.json", type=click.Path(exists=True))
 def disable(filename):
     """ Disable counter configuration in config_db file """
-    _update_config_db("disable", filename)
+    _update_config_db_flex_counter_table("disable", filename)
 
+@config_db.command()
+@click.argument("filename", default="/etc/sonic/config_db.json", type=click.Path(exists=True))
+def delay(filename):
```

Comment:
@shlomibitton do we need a command for the opposite action? Like we have for config-db enable/disable?

Reply:
I don't think so, this is basically only for fast-reboot purposes when we want to delay the counters creation, so the only use for this is before fast-reboot.

Reply:
@shlomibitton ok